doc/design/send_recv.md

```diff
@@ -0,0 +1,68 @@
+# Send and Recv OP
+
+With PaddlePaddle Fluid, the node to node communication is done
+through the send and recv OP. The send and recv OP will be
+automatically added to
+the
+[`ProgramDesc`](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/program.md#blockdesc-and-programdesc) by
+the
+[distributed transpiler](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/dist_refactor/distributed_architecture.md#distributed-transpiler) when
+a local training `ProgramDesc` is transpiled to the distributed
+training `ProgramDesc`.
+
+
+## Send OP
+
+The send OP sends its input tensors to one or multiple recv OPs and
```

Comment:
It seems that Send is a broadcasting operation, as you said it could send tensors to more than one recv operators? Is a broadcasting send operator what we want?